specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-08-01/managedClusters.json

```diff
@@ -3868,6 +3868,71 @@
       "title": "A time range.",
       "description": "For example, between 2021-05-25T13:00:00Z and 2021-05-25T14:00:00Z."
     },
+    "ScheduledEvent": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of a scheduled event."
+        },
+        "eventid": {
+          "type": "string",
+          "description": "The event id of a scheduled event."
+        },
+        "eventsource": {
+          "type": "string",
+          "description": "The source of a scheduled event."
+        },
+        "eventstatus": {
+          "type": "string",
+          "enum": [
+            "Scheduled",
+            "Started",
+            "Completed",
+            "Cancelled"
```

Comment:
Failed?

Reply:
in the message we sent, we will not know the status of maintenance event, we just tell cx what will happen.

these statuses are used in vmss events, I think it is probably better to be align with them too.